.github/workflows/build_library.yml

```diff
@@ -47,10 +47,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - macos-version: "macos-10.15"
-            target: x86_64-apple-darwin
-            py-platform: macosx-10_15_x86_64
-          - macos-version: "macos-11.0"
+          - macos-version: "macos-11"
```

Comment:
I checked GitHub's recommendation to see if there was any difference between `macos-11` and `macos-11.0`, and they recommend the former (the latter may not even exist). See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources.